recipes/gnutls/3.6.14/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -ex
+
+n="$CPU_COUNT"
+PKG_CONFIG="/usr/local/pkg-config-0.28"
```

Comment:
Is this going to be there on the build host?

Reply:
No, that probably shouldn't included.